[Bug]: test_frontends/config/tensorflow:isscalar is not handling tuple properly

### Bug Explanation

The `isscalar` function from `test_frontends/config/tensorflow` causes the tests to fail if it's called with a tuple as input argument.

Here is the log from a test failure:
```
FAILED [ 40%]
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:2116 (test_tensorflow_unique[cpu-jax-False-False])
@handle_frontend_test(
>       fn_tree="tensorflow.unique",
        dtype_x=helpers.dtype_and_values(
            available_dtypes=["int64", "int32"],
            min_value=1,
            max_value=100,
            min_dim_size=1,
            max_dim_size=10,
            min_num_dims=1,
            max_num_dims=1,
        ),
        test_with_out=st.just([False]),
    )

test_general_functions.py:2118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test_general_functions.py:2140: in test_tensorflow_unique
    helpers.test_frontend_function(
../../helpers/function_testing.py:813: in test_frontend_function
    if frontend_config.isscalar(frontend_ret):
../config/tensorflow.py:106: in isscalar
    return tf.experimental.numpy.isscalar(x)
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_array_ops.py:989: in isscalar
    return ndim(num) == 0
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_array_ops.py:983: in ndim
    a = asarray(a)
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_array_ops.py:219: in asarray
    return array(a, dtype, copy=False)
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_array_ops.py:206: in array
    return _array_internal(val, dtype, copy, ndmin)
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_array_ops.py:165: in _array_internal
    result_t = np_arrays.convert_to_tensor(result_t, dtype_hint=dtype)
/opt/fw/tensorflow/tensorflow/python/ops/numpy_ops/np_arrays.py:46: in convert_to_tensor
    return tensor_conversion.convert_to_tensor_v2_with_dispatch(
/opt/fw/tensorflow/tensorflow/python/util/traceback_utils.py:153: in error_handler
    raise e.with_traceback(filtered_tb) from None
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

e = _NotOkStatusException(), name = 'packed'

    def raise_from_not_ok_status(e, name):
      e.message += (" name: " + str(name if name is not None else ""))
>     raise core._status_to_exception(e) from None  # pylint: disable=protected-access
E     tensorflow.python.framework.errors_impl.InvalidArgumentError: cannot compute Pack as input #1(zero-based) was expected to be a int64 tensor but is a int32 tensor [Op:Pack] name: packed
E     Falsifying example: test_tensorflow_unique(
E         frontend='tensorflow',
E         backend_fw='jax',
E         on_device='cpu',
E         dtype_x=(['int64'], [array([0])]),
E         test_flags=FrontendFunctionTestFlags(
E             num_positional_args=0,
E             with_out=[False],
E             inplace=False,
E             as_variable=[False],
E             native_arrays=[False],
E             generate_frontend_arrays=False,
E         ),
E         fn_tree='ivy.functional.frontends.tensorflow.unique',
E     )
E     
E     You can reproduce this example by temporarily adding @reproduce_failure('6.82.0', b'AAAAAAAAAAAAAA==') as a decorator on your test case

/opt/fw/tensorflow/tensorflow/python/framework/ops.py:6656: InvalidArgumentError

```

### Steps to Reproduce Bug

Adding `@reproduce_failure('6.82.0', b'AAAAAAAAAAAAAA==')` to `test_tensorflow_unique` from `test_general_functions` for example.


### Environment

MacOS 13.3.1 using PyCharm Professional and Remote Python 3.10.0 Docker interpreter

### Ivy Version

v1.1.9

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [ ] JAX

### Device

CPU

--- a/ivy_tests/test_ivy/test_frontends/config/tensorflow.py
+++ b/ivy_tests/test_ivy/test_frontends/config/tensorflow.py
@@ -103,4 +103,4 @@ def as_native_dev(device: str):
 
 
 def isscalar(x):
-    return tf.experimental.numpy.isscalar(x)
+    return is_native_array(x) and tf.experimental.numpy.isscalar(x)

